node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
+  "systemParams": "darwin-arm64-102",
+  "modulesFolders": [],
+  "flags": [],
+  "linkedModules": [],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```

Comment:
Where did this come from?

Reply:
Hmm, indeed. My quick-finger `git add -A` missed that one. I will remove it.